concepticondata/conceptlists/Lundgren-2020-275.tsv

```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-132	132	132	HIGH UP (ADV.)	1265	HIGH
+Lundgren-2020-275-133	133	133	HOLE	1667	HOLE
+Lundgren-2020-275-134	134	134	HORN	1393	HORN (ANATOMY)
+Lundgren-2020-275-135	135	135	HORSEFLY	3262	HORSEFLY
+Lundgren-2020-275-136	136	136	HOUSE	1252	HOUSE
+Lundgren-2020-275-137	137	137	HOWLER MONKEY	433	HOWLER MONKEY
+Lundgren-2020-275-138	138	138	HUNGRY	1533	BE HUNGRY
+Lundgren-2020-275-139	139	139	HURT		
+Lundgren-2020-275-140	140	140	HUSBAND	1200	HUSBAND
+Lundgren-2020-275-141	141	141	INDIGENOUS PERSON	3612	INDIGENOUS PEOPLES OF THE AMERICAS
+Lundgren-2020-275-142	142	142	INVITE (TR.)	1611	INVITE
+Lundgren-2020-275-143	143	143	ISLAND	622	ISLAND
+Lundgren-2020-275-144	144	144	JOIN~ATTACH	1917	TIE
+Lundgren-2020-275-145	145	145	KILL	1417	KILL
+Lundgren-2020-275-146	146	146	KNIFE	1352	KNIFE
+Lundgren-2020-275-147	147	147	LAKE	624	LAKE
+Lundgren-2020-275-148	148	148	LAND	626	LAND
+Lundgren-2020-275-149	149	149	LARVA	2640	LARVA
+Lundgren-2020-275-150	150	150	LATE	477	LATE
+Lundgren-2020-275-151	151	151	LEAVE (TR.)		
+Lundgren-2020-275-152	152	152	LEAVE FROM (INTR.)	1752	LEAVE
+Lundgren-2020-275-153	153	153	LIVER~HEART		
+Lundgren-2020-275-154	154	154	LONG	1203	LONG
+Lundgren-2020-275-155	155	155	LONG TIME AGO	373	LONG AGO
+Lundgren-2020-275-156	156	156	LOOK (INTR.)	1819	LOOK
+Lundgren-2020-275-157	157	157	LOOK FOR (TR.)	1468	LOOK FOR
+Lundgren-2020-275-158	158	158	LOSE WAY		
+Lundgren-2020-275-159	159	159	LOSE ONESELF		
+Lundgren-2020-275-160	160	160	LOUSE	1392	LOUSE
+Lundgren-2020-275-161	161	161	LUNG~BREATHE (INTR.)		
+Lundgren-2020-275-162	162	162	MACAW	1630	MACAW
+Lundgren-2020-275-163	163	163	MANIOC FLOUR	324	MANIOC FLOUR
+Lundgren-2020-275-164	164	164	MANY~MUCH	1198	MANY
+Lundgren-2020-275-165	165	165	MIX (TR.)~PRESS		
+Lundgren-2020-275-166	166	166	MONKEY	1350	MONKEY
+Lundgren-2020-275-167	167	167	MOON	1313	MOON
+Lundgren-2020-275-168	168	168	MOSQUITO	1509	MOSQUITO
+Lundgren-2020-275-169	169	169	MOURN	1955	GRIEVE
+Lundgren-2020-275-170	170	170	MOUSE~RAT	2139	MUROID (MOUSE OR RAT)
+Lundgren-2020-275-171	171	171	MOUTH	674	MOUTH
+Lundgren-2020-275-172	172	172	NAIL (BODY PART)	1896	FINGERNAIL OR TOENAIL
+Lundgren-2020-275-173	173	173	NAME		
+Lundgren-2020-275-174	174	174	NECK~THROAT		
+Lundgren-2020-275-175	175	175	NEPHEW	1063	NEPHEW
+Lundgren-2020-275-176	176	176	NEW	1231	NEW
+Lundgren-2020-275-177	177	177	NIGHT	1233	NIGHT
+Lundgren-2020-275-178	178	178	NOSE	1221	NOSE
+Lundgren-2020-275-179	179	179	OBEY (TR.)	1635	OBEY
+Lundgren-2020-275-180	180	180	OPEN (TR.)	1156	OPEN
+Lundgren-2020-275-181	181	181	OTHER	197	OTHER
+Lundgren-2020-275-182	182	182	OWNER	1036	OWNER
+Lundgren-2020-275-183	183	183	PACA	316	PACA
+Lundgren-2020-275-184	184	184	PADDLE (N.)	234	PADDLE
+Lundgren-2020-275-185	185	185	PAINT~WRITE		
+Lundgren-2020-275-186	186	186	PATH	2252	PATH
+Lundgren-2020-275-187	187	187	PATIO		
+Lundgren-2020-275-188	188	188	PEANUT	2364	PEANUT
+Lundgren-2020-275-189	189	189	PERSON	683	PERSON
+Lundgren-2020-275-190	190	190	PET		
```

Comment:
This (or something like domesticated animal) could perhaps be an interesting concept to add as there is already the counterpart in Concepticon, WILD ANIMAL.

Reply:
Good point, I'll add then PET to concepticon.tsv.